src/lib/create.c

```diff
@@ -301,7 +301,7 @@ hash_bn(pgp_hash_t *hash, BIGNUM *bignum)
 static int
 hash_key_material(const pgp_seckey_t *key, uint8_t *result)
 {
-    pgp_hash_t hash;
+    pgp_hash_t hash = {0};
```

Comment:
@dewyatt you're right and I've checked supported distributives and all of them has compiler that works. I expect only one case when `{0}` doesn't work — some tolchain for embedded development but I think it is out of scope.